src/Apps/Sell/Components/BottomFormNavigation.tsx

```diff
@@ -1,28 +1,54 @@
-import { Box, Button, Flex, Link } from "@artsy/palette"
+import { Box, Button, Flex } from "@artsy/palette"
 import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
 import { useRouter } from "System/Router/useRouter"
 import { useFormikContext } from "formik"
+import { useState } from "react"
 
 export const BottomFormNavigation = () => {
-  const { isValid, isSubmitting, submitForm } = useFormikContext()
+  const { isValid, submitForm } = useFormikContext()
   const { router } = useRouter()
   const {
     actions,
     state: { isFirstStep, isLastStep, submissionID },
   } = useSellFlowContext()
 
-  const onContinue = async () => {
-    await submitForm()
+  const [isLoadingBack, setIsLoadingBack] = useState(false)
+  const [isLoadingContinue, setIsLoadingContinue] = useState(false)
 
-    actions.goToNextStep()
+  const onBack = async () => {
+    setIsLoadingBack(true)
+
+    try {
+      await submitForm()
+
+      actions.goToPreviousStep()
+    } catch (error) {
+      console.error("Error submitting form", error)
```

Comment:
Todo: Use `logger` instead:

```
const logger = createLogger("BottomFormNavigation.tsx")
...
logger.error("Error submitting form, error) 
```


https://artsy.slack.com/archives/C05EQL4R5N0/p1718379105202439

Reply:
We should probably update the logger everywhere in the new submission flow.